hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java

```diff
@@ -618,8 +618,7 @@ private void selectStreamingInputStreams(
     Map<AsyncLogger, RemoteEditLogManifest> resps =
         loggers.waitForWriteQuorum(q, selectInputStreamsTimeoutMs,
             "selectStreamingInputStreams");
-
-    LOG.debug("selectStreamingInputStream manifests:\n" +
+    LOG.debug("selectStreamingInputStream manifests:\n {}",
```

Comment:
This one still needs wrapped in `if (LOG.isDebugEnable)` otherwise the `Joiner` will always be evaluated before the parameters are passed to the logger.

Reply:
Thanks for review.I hava fixed it in the latest commit.